src/server/v2.0/handler/gc.go

```diff
@@ -30,13 +36,19 @@ func newGCAPI() *gcAPI {
 }
 
 func (g *gcAPI) Prepare(ctx context.Context, operation string, params interface{}) middleware.Responder {
-	if err := g.RequireSysAdmin(ctx); err != nil {
-		return g.SendError(ctx, err)
+	secCtx, ok := security.FromContext(ctx)
+	if !ok {
+		return g.SendError(ctx, errors.UnauthorizedError(errors.New("security context not found")))
 	}
+	g.secCtx = secCtx
+	g.resource = system.NewNamespace().Resource(rbac.ResourceGarbageCollection)
```

Comment:
We can't assign the var to the gcAPI because the instance of gcAPI is shared for all API request.

Reply:
Suggest to add a method `RequireSystemAccess` in the BaseAPI to handle the system permission checking.

Reply:
Because now the operations of the GC use different permission actions so I think we need to remove the permission checking in the Prepare. Every operation uses the `RequireSystemAccess` to check permission.

Reply:
do no know this either, ok will have a update.